mlir/test/Integration/Dialect/Vector/CPU/ArmSME/test-ssve.mlir

```diff
@@ -2,7 +2,7 @@
 // DEFINE: %{compile} = mlir-opt %s -test-lower-to-llvm
 // DEFINE: %{run} = %mcr_aarch64_cmd \
 // DEFINE:  -march=aarch64 -mattr=+sve,+sme \
-// DEFINE:  --force-streaming-compatible-sve \
+// DEFINE:  -force-streaming-compatible \
```

Comment:
Why does it need this flag? Is the `arm_streaming` attribute not propagated to the LLVM IR function?

Reply:
Yeah it's a good point, think this was the first test we added and the function attribute hadn't been wired up yet or something. Updated to remove the flag altogether.